reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.        res[offset_Z+ind_large] = 1 - np.sum(coverages)
</pre><p>Does this need to exclude the largest species from the sum?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Nope. It is solving the equation <code>sum (Yk) = 1</code>. Hence all species fractions must be included.</p>